app/controllers/posts_controller.rb

```diff
@@ -13,4 +13,27 @@ def show
       @post = nil
     end
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
```

Comment:
- [ ] Good job adding forms 👍 , however, we still need to fix some points 
Let's create users, `first_user: Tom` and `second_user: Lilly`, then let's create a post for all users `tom's 1 post` and `lilly's 1 post`, but what we are inspecting at the output, is only `first_user: Tom` has gotten all posts.

![Screenshot 2023-08-25 144146](https://github.com/yosaddis/blog-app-rails/assets/57408419/42a140aa-2ff2-4ca2-b8d3-17953f656668)

And `Lilly` has 0 post

![Screenshot 2023-08-25 151454](https://github.com/yosaddis/blog-app-rails/assets/57408419/b1977ab0-134e-485f-9339-f284e6e6a8ac)


Kindly fix this 😄

Reply:
Dear  @joel-kalema  Thank you for reviewing.
The above issue is happening due to the requirement of the project. We can't associate the current_user to the user creating the post. This will be implemented in the next steps of the project. Kindly see the screenshot. 
![image](https://github.com/yosaddis/blog-app-rails/assets/60548252/4618da3b-382a-4831-a71e-42569593030f)